apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -651,4 +652,36 @@ export class ApplicationController {
 
     return application
   }
+
+  @Put('application/:id/presignedUrl')
+  @ApiParam({
+    name: 'id',
+    type: String,
+    required: true,
+    description: 'The id of the application to update the state for.',
```

Comment:
The description is not very related to the function it's doing. I agree a lot of other endpoints are also not well documented

Reply:
Correct! :D Will fix :)

Reply:
I am going to update the description as well on other endpoints I have created, hope you don't mind adding it to this PR 😊

Reply:
no problem 👍